src/components/AddItemForm/AddItemForm.js

```diff
@@ -2,15 +2,24 @@ import { useState } from 'react';
 import { db } from '../../lib/firebase.js';
 
 const AddItemForm = ({ listId }) => {
+  /** Default Values **/
   const defaultFormValues = {
     itemName: '',
     purchaseInterval: '7',
   };
 
+  const defaultErrorMessage = '';
```

Comment:
I'm curious why the defaultErrorMessage variable if it's just for setting the error to an empty string? It was a little counterintuitive to me when reading further down in the code. For example when I see `setErrorMessage(defaultErrorMessage);`, I'm not immediately clear on what you are doing unless I refer back to the line where you declare the variable. Whereas if you just did `setErrorMessage('');` I'd know without looking up from that line exactly what was going on.

Reply:
I'm also wondering here if it might be easier to set this state item as an empty string? So you would have: 

`const [errorMessage, setErrorMessage] = useState('');`

Then: 

`const handleChange = (event) => {
    setErrorMessage(''); // if input field changes, reset error message`

Reply:
We were debating whether to follow what we had did for defaultFormValues, but this is certainly cleaner! 

We've pushed a commit to simply reset the error message to an empty string.

https://github.com/the-collab-lab/tcl-26-smart-shopping-list/pull/28/commits/99db234ffd95649484cb53d100defa57353c92f1